src/panels/CharmsPanel/CharmApplicationsDetails/CharmApplicationsDetails.tsx

```diff
@@ -23,19 +27,23 @@ const CharmApplicationsDetails = ({ charmURL }: Props): JSX.Element => {
 
   return (
     <p className="p-form-help-text is-tick-element">
-      <Tooltip
-        message={
-          hiddenApplications.length ? (
-            <span>..., {formatApplicationsDetails(hiddenApplications)}</span>
-          ) : null
-        }
-        position="btm-left"
-      >
-        {formatApplicationsDetails(displayedApplications)}
-        {hiddenApplications.length ? (
-          <span>, +{hiddenApplications.length} more...</span>
-        ) : null}
-      </Tooltip>
+      {formatApplicationsDetails(displayedApplications)}
+      {hiddenApplications.length ? (
+        <>
+          {" + "}
+          <Tooltip
+            tooltipClassName="p-tooltip--fixed-x-small-width"
+            message={
+              isSmallScreen
+                ? formatApplicationsDetails(hiddenApplications)
+                : breakLines(formatApplicationsDetails(hiddenApplications))
+            }
```

Comment:
I've suggested a small change to the CSS that means we don't have to check for the screen size in this component and conditionally break the lines.

```suggestion
            message={breakLines(formatApplicationsDetails(hiddenApplications))}
```